care/facility/api/serializers/file_upload.py

```diff
@@ -93,6 +93,8 @@ def check_permissions(file_type, associating_id, user, action="create"):
             return associating_id
         elif file_type == FileUpload.FileType.COMMUNICATION.value:
             return associating_id
+        elif file_type == FileUpload.FileType.CONSENT_RECORD.value:
```

Comment:
use this block instead, as access achecks are needed
https://github.com/coronasafe/care/blob/ce7073e4d789dd19272d257cc79c2500d3d53e75/care/facility/api/serializers/file_upload.py#L35-L36

Reply:
made changes